packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -62,6 +62,7 @@ export interface ISocketStorageDiscovery {
      * The AFD URL for PushChannel
      */
     deltaStreamSocketUrl2?: string;
+    socketToken: string;
```

Comment:
I assume it needs to be optional, no?

Reply:
I think it is always returned but can be empty, making it optional since having empty token is confusing